app-libs/exchange-oracle/src/coingecko.rs

```diff
@@ -33,7 +39,15 @@ const COINGECKO_PARAM_CURRENCY: &str = "vs_currency";
 const COINGECKO_PARAM_COIN: &str = "ids";
 const COINGECKO_PATH: &str = "api/v3/coins/markets";
 const COINGECKO_TIMEOUT: Duration = Duration::from_secs(3u64);
-
+//TODO from API ?
+lazy_static! {
+	static ref SYMBOL_ID: HashMap<&'static str, &'static str> = HashMap::from([
```

Comment:
Is that a name that is frequently used for currency IDs? Otherwise, I'd suggest using a more descriptive name. For me, a symbol is very general.

Reply:
Symbol and Id are the name used by Coingecko API. Ex: Corresponding id to symbol "DOT" is "polkadot". This map is here to map the symbol (general) into the id required by Coingecko API. Most likely Coingecko specific. 
I will rename the map to SYMBOL_ID_MAP